### ISSUE TITLE
Queue list limit?

### ISSUE THREAD

**Issue by dubiousdavid** [taskforcesh/issues#51]
I'm not seeing all of my queues. Is there a limit?

**Comment by manast** [comment]
There is no limit. Is there anything special with the queue names, like prefixes or similar?

**Comment by dubiousdavid** [comment]
No. Nothing special about the queue names compared to the rest of the queues. They all have the same prefix.

**Comment by manast** [comment]
And how many queues are there in total?

**Comment by dubiousdavid** [comment]
I'm not 100% sure how many there should be. It's definitely less than 50, though.
